node/src/main/scala/org/bitcoins/node/networking/peer/DataMessageHandler.scala

```diff
@@ -391,6 +382,18 @@ case class DataMessageHandler(
               }
             }
 
+        getHeadersF.recoverWith {
+          case _: DuplicateHeaders =>
+            logger.debug(
```

Comment:
It might be worth `warn` here, as we don't want to see this happen too much right?